test-projects/gtm-test/src/pages/index.js

```diff
@@ -14,7 +14,7 @@ const IndexPage = () => (
     <div style={{ maxWidth: `300px`, marginBottom: `1.45rem` }}>
       <Image />
     </div>
-    <Link to="/page-2/">Go to page 2</Link>
+    <Link to="/page-2/" className="page-2-link">Go to page 2</Link>
```

Comment:
thought: Removing the class.

In general, as much as we can, we should leverage the same things the user would do. The user would click on the "Go to page 2" link, so we should retrieve it with `cy.contains('Go to page 2')`.